### ISSUE TITLE
the generated audio are nothing

### ISSUE THREAD

**Issue by hujinsen** [deNsuh/segan-pytorch#30]
the waveform of the denoised audio are zero.
![image](https://user-images.githubusercontent.com/6258078/58770787-8364c300-85e3-11e9-8678-247edff1cf8c.png)

**Comment by dansuh17** [comment]
Producing zero samples is most likely a symptom of discriminator failing. Have you checked the loss graph of discriminator?

**Comment by SureArtificialIntelligence** [comment]
Hi, did you solve this problem? I met the same problem at the beginning of training

**Comment by SureArtificialIntelligence** [comment]
The groundtruth is the raw audio, but the generated audio is range [-1, 1] due to Tanh() .......

The code does not work without modification
